analyses/independent-samples/00-repeated-samples.Rmd

```diff
@@ -33,20 +33,21 @@ set.seed(201910)
 root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
 if(params$base_run == 0) {
 sample_df <- readr::read_tsv(file.path(root_dir, "data", "histologies.tsv"), guess_max = 10000)
-} else {
-sample_df <- readr::read_tsv(file.path(root_dir, "data", "histologies-base.tsv"), guess_max = 10000)
-}
+} 
+#else {
+#sample_df <- readr::read_tsv(file.path(root_dir, "data", #"histologies-base.tsv"), guess_max = 10000)
+#}
```

Comment:
Can we keep this code ? Because`histologies-base.tsv` file is used as a input file when we do cancer subtyping which we add to back as a update to make the final `histologies.tsv` file for data release.

Reply:
Done